test/tests/api/v1_1/nodes_tests.py

```diff
@@ -106,9 +106,11 @@ def get_undiscovered_compute_count(self):
                 workflows = self.__get_data()
                 LOG.info(workflows)
                 for workflow in workflows:
-                    if workflow.get['injectableName'] == 'Graph.SKU.Discovery' \
-                     and workflow.get['status'] == 'succeeded':
-                     discovered_count += 1
+                    definition = workflow.get('definition', {})
+                    injectableName = definition.get('injectableName')
+                    if injectableName == 'Graph.SKU.Discovery':
+                        workflow.get['_status'] == 'succeeded':
```

Comment:
a succeeded Graph.SKU.Discovery workflow doesn't means the node has been discovered successfully?